client/src/modules/dashboard/Events/pages/EventPage.tsx

```diff
@@ -90,11 +90,7 @@ export const EventPage: NextPage = () => {
           <h2>Venue: Online</h2>
         )}
       </Box>
-      {data.event.sponsors.length ? (
```

Comment:
I'm curious why this was removed? Do we not want to show the sponsors on the event page - or is there some other implementation in mind or something?

Reply:
@moT01, This removal is not intentional. This somehow got added to the staged files. Can I remove it  here?

Reply:
@moT01 , I've added back the code.